src/Countries/Nigeria.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Nigeria extends Country
+{
+    public function countryCode(): string
+    {
+        return 'ng';
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'New Year\'s Day' => "01-01",
+            'Worker\'s Day' => "05-01",
+            'Children\'s Day' => "05-27",
```

Comment:
Is this an official public holiday? I don't find it here: https://publicholidays.africa/nigeria/2024-dates/

Reply:
I guess it's not regarded as an official holiday, i'll remove it @Nielsvanpach

Reply:
There also seem to be official Muslim holidays on the list. I'm not aware of the rules in Nigeria on this? Is this a public holiday for everyone?

Reply:
The muslim holidays are for everyone but the exact calculation of these holidays involves the sighting of the moon, and it can vary depending on different interpretations within the Islamic community. @Nielsvanpach